pyscriptjs/src/main.ts

```diff
@@ -272,7 +305,10 @@ from pyscript import micropip, Element, console, document`);
                 const filename = pathArr.pop();
                 // TODO: Would be probably be better to store plugins somewhere like /plugins/python/ or similar
                 const destPath = `./${filename}`;
-                await runtime.loadFromFile(destPath, singleFile);
+                const response = await robustFetch(singleFile);
+                const buffer = await response.arrayBuffer();
+                const data = new Uint8Array(buffer);
+                runtime.writeToFile(destPath, data);
```

Comment:
This seems a duplication of the above logic, what do you think about creating a helper method for it? 🤔